src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -336,7 +335,6 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setAllowedTypes('choice_list', array('null', 'Symfony\Component\Form\ChoiceList\ChoiceListInterface'));
         $resolver->setAllowedTypes('choices', array('null', 'array', '\Traversable'));
         $resolver->setAllowedTypes('choice_translation_domain', array('null', 'bool', 'string'));
-        $resolver->setAllowedTypes('choices_as_values', 'bool');
```

Comment:
Why not use `array('null', 'bool')` then (well okay, technically it does not make much of a difference as the user would get an error anyway)?